### ISSUE TITLE
Issue with using pysph view

### ISSUE THREAD

**Issue by Phanindrasp** [pypr/pysph#346]
I'm a beginner to pysph. I managed to install the package along with all the dependencies. Using the command "pysph run dam_break_2d" works for me and even prints the output data in the appropriate location. However, when I use  pysph view dam_break_2d_output/, it throws up the following error:

I'm using mpirun (Open MPI) 2.1.1.

It looks like orte_init failed for some reason; your parallel process is
likely to abort.  There are many reasons that a parallel process can
fail during orte_init; some of which are due to configuration or
environment problems.  This failure appears to be an internal failure;
here's some additional information (which may only be relevant to an
Open MPI developer):

  setting topology failed
  --> Returned value Error (-1) instead of ORTE_SUCCESS
--------------------------------------------------------------------------
--------------------------------------------------------------------------
It looks like orte_init failed for some reason; your parallel process is
likely to abort.  There are many reasons that a parallel process can
fail during orte_init; some of which are due to configuration or
environment problems.  This failure appears to be an internal failure;
here's some additional information (which may only be relevant to an
Open MPI developer):

  orte_ess_init failed
  --> Returned value Error (-1) instead of ORTE_SUCCESS
--------------------------------------------------------------------------
--------------------------------------------------------------------------
It looks like MPI_INIT failed for some reason; your parallel process is
likely to abort.  There are many reasons that a parallel process can
fail during MPI_INIT; some of which are due to configuration or environment
problems.  This failure appears to be an internal failure; here's some
additional information (which may only be relevant to an Open MPI
developer):

  ompi_mpi_init: ompi_rte_init failed
  --> Returned "Error" (-1) instead of "Success" (0)
--------------------------------------------------------------------------
*** An error occurred in MPI_Init_thread
*** on a NULL communicator
*** MPI_ERRORS_ARE_FATAL (processes in this communicator will now abort,
***    and potentially your MPI job)
[phani-B85M-DS3H:7967] Local abort before MPI_INIT completed completed successfully, but am not able to aggregate error messages, and not able to guarantee that all other processes were killed!

Any direction to debug this issue will be greatly appreciated :)

**Comment by prabhuramachandran** [comment]
I am not clear as to what you are doing here.  Are you using `mpirun` for `pysph view`?  If so `pysph view` should be run as such without `mpirun/mpiexec`.  The viewer runs as a single process without MPI.

**Comment by Phanindrasp** [comment]
No, I was just using the command "pysph view dam_break_2d_output/" or "pysph view" without mpirun. Not sure why this error pops up though.
